Rakefile

```diff
@@ -4,3 +4,45 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+namespace :unicorn do
+  desc "Start unicorn for development env."
+  task(:start) {
```

Comment:
__[rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)